src/stylesheets/HomepageRow.css

```diff
@@ -64,6 +64,10 @@ p {
   padding-left: 10px;
 }
 
+/* ============================================= */
+/*              Responsive Browsing              */
+/* ============================================= */
+
```

Comment:
Please remove this comment since this commenting style has not been established in other parts of the app 👍